packages/cli/src/lib/commands/collect/index.ts

```diff
@@ -1,12 +1,12 @@
-import {YargsCommandObject} from '../../core/yargs/types';
-import {logVerbose} from '../../core/loggin/index';
-import {COLLECT_OPTIONS} from './options';
-import {runCollectCommand} from "./command-impl";
+import { YargsCommandObject } from '../../core/yargs/types';
+import { logVerbose } from '../../core/loggin';
+import { collectOptions } from './options';
+import { runCollectCommand } from './command-impl';
 
 export const collectUserFlowsCommand: YargsCommandObject = {
   command: 'collect',
   description: 'Run a set of user flows and save the result',
-  builder: (y) => y.options(COLLECT_OPTIONS),
+  builder: (y) => y.options(collectOptions),
```

Comment:
I question my desition on changing the naming convention to not be CONST, it may be better to maintain the old names.